radio/src/bluetooth.cpp

```diff
@@ -64,6 +64,7 @@ char * bluetoothReadline()
 
   while (1) {
     if (!btRxFifo.pop(byte)) {
+      TRACE("NO RESPONSE FROM BT MODULE");
```

Comment:
It did not att all on X9E, only when sending a CLI command to module and module would not respond. Will do some tests with Horus